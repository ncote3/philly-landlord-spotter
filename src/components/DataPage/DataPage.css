.DataPageContainer {
    margin-top: 6vw !important;
    text-align: left;
    margin-bottom: 3vw;
}

.dataDescCardHeader {
    background-color: #343a40 !important;
    color: white !important;
}

.dataDescCardBold {
    font-weight: bold;
}

.dataDescCardItalic {
    font-style: italic;
}
