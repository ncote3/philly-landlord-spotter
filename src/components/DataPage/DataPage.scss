.DataPageWrapper {
  padding-top: 6vw !important;
  text-align: left;
  padding-bottom: 3vw;
  background-attachment: fixed;
  background-position: right top;
  width: 100%;
  height: auto;
  .DataPageContainer {
    .DataPageSection {
      background-color: rgba(0, 0, 0, 0.85);
      box-shadow: 0 0.35vw 0.35vw 0 rgba(0, 0, 0, 0.6);
      padding: 1.5vw 1.5vw 0.5vw;
      margin-bottom: 2rem;
    }
    .DataPageJumbotron {
      background-color: rgba(0, 0, 0, 0.85);
      box-shadow: 0 0.35vw 0.35vw 0 rgba(0, 0, 0, 0.6);
      color: #fff;
      padding: 2rem 1rem;
      margin-bottom: 2rem;
      border-radius: 0.3rem;
    }
    .whiteText {
      color: #fff;
    }
  }
}

@media (max-width: 1024px) {
  .DataPageWrapper {
    padding-top: 8vw !important;
  }
}

@media (max-width: 768px) {
  .DataPageWrapper {
    padding-top: 10vw !important;
  }
}

@media (max-width: 591px) {
  .DataPageWrapper {
    padding-top: 12vw !important;
  }
}

@media (max-width: 450px) {
  .DataPageWrapper {
    padding-top: 15vw !important;
  }
}

@media (max-width: 414px) {
  .DataPageWrapper {
    padding-top: 16vw !important;
  }
}

@media (max-width: 375px) {
  .DataPageWrapper {
    padding-top: 18vw !important;
  }
}
