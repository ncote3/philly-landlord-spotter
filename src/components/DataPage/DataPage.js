import React from "react";
import Jumbotron from 'react-bootstrap/Jumbotron'
import Container from "react-bootstrap/Container";
import Card from 'react-bootstrap/Card';
import CardColumns from 'react-bootstrap/CardColumns'
import './DataPage.css';

export default function DataPage() {
    return (
        <div className='DataPageContainer'>
            <Container>
                <Jumbotron>
                    <h1>Data</h1>
                    <p>An explanation on where this data came from and what it contains.</p>
                </Jumbotron>
                <div>
                    <h1>Data Source</h1>
                    <p>
                        In 2012 the City of Philadelphia began pursuing and enacting a public open data program through
                         <a href='https://www.opendataphilly.org/'> OpenDataPhilly</a>. The data used in this project is provided
                        by the aforementioned program, in their
                        <a href='https://www.opendataphilly.org/dataset/opa-property-assessments'> Property Assessment </a>
                         dataset. Publicly available, anyone can view or use the dataset without barrier.
                    </p>
                </div>
                <div>
                    <h1>Data Description</h1>
                    <p>
                        This dataset is comprised on completely public information. It does not contain any personal
                        information about who lives in a property or the vacancy status of the properties.
                        <br/>
                        The data used for this site contains the following for each property listed:
                    </p>
                    <div>
                        <CardColumns>
                            <Card border={'dark'}>
                                <Card.Header className='dataDescCardBold dataDescCardHeader'>Owner 1 Name</Card.Header>
                                <Card.Body>
                                    <Card.Text>
                                        <p>
                                            <span className={'dataDescCardBold'}>Example: </span>
                                            <span className={'dataDescCardItalic'}>John Doe</span>
                                        </p>
                                        <p>
                                            <span className={'dataDescCardBold'}>Description:</span> The primary owner
                                            of a property.
                                        </p>
                                        <p>
                                            <span className={'dataDescCardBold'}>Data Reference: </span>
                                            <span className={'dataDescCardItalic'}>owner_1</span>
                                        </p>
                                    </Card.Text>
                                </Card.Body>
                            </Card>
                            <Card border={'dark'}>
                                <Card.Header className='dataDescCardBold dataDescCardHeader'>Owner 2 Name</Card.Header>
                                <Card.Body>
                                    <Card.Text>
                                        <p>
                                            <span className={'dataDescCardBold'}>Example: </span>
                                            <span className={'dataDescCardItalic'}>Jane Doe</span>
                                        </p>
                                        <p>
                                            <span className={'dataDescCardBold'}>Description:</span> The secondary owner
                                            of a property. This is not always provided.
                                        </p>
                                        <p>
                                            <span className={'dataDescCardBold'}>Data Reference: </span>
                                            <span className={'dataDescCardItalic'}>owner_2</span>
                                        </p>
                                    </Card.Text>
                                </Card.Body>
                            </Card>
                            <Card border={'dark'}>
                                <Card.Header className='dataDescCardBold dataDescCardHeader'>Street Address </Card.Header>
                                <Card.Body>
                                    <Card.Text>
                                        <p>
                                            <span className={'dataDescCardBold'}>Example: </span>
                                            <span className={'dataDescCardItalic'}>123 Street Road</span>
                                        </p>
                                        <p>
                                            <span className={'dataDescCardBold'}>Description:</span> The street address
                                            of the property.
                                        </p>
                                        <p>
                                            <span className={'dataDescCardBold'}>Data Reference: </span>
                                            <span className={'dataDescCardItalic'}>location</span>
                                        </p>
                                    </Card.Text>
                                </Card.Body>
                            </Card>
                            <Card border={'dark'}>
                                <Card.Header className='dataDescCardBold dataDescCardHeader'>Geocoordinates  </Card.Header>
                                <Card.Body>
                                    <Card.Text>
                                        <p>
                                            <span className={'dataDescCardBold'}>Example: </span>
                                            <span className={'dataDescCardItalic'}>[40.003088,-75.108357]</span>
                                        </p>
                                        <p>
                                            <span className={'dataDescCardBold'}>Description:</span> The latitude and
                                            longitude of a property.
                                        </p>
                                        <p>
                                            <span className={'dataDescCardBold'}>Data Reference: </span>
                                            <span className={'dataDescCardItalic'}>lat lng</span>
                                        </p>
                                    </Card.Text>
                                </Card.Body>
                            </Card>
                            <Card border={'dark'}>
                                <Card.Header className='dataDescCardBold dataDescCardHeader'>Sale Date  </Card.Header>
                                <Card.Body>
                                    <Card.Text>
                                        <p>
                                            <span className={'dataDescCardBold'}>Example: </span>
                                            <span className={'dataDescCardItalic'}>2013-02-24</span>
                                        </p>
                                        <p>
                                            <span className={'dataDescCardBold'}>Description:</span> The last recorded
                                            sale of the property, in Year-Month-Day format.
                                        </p>
                                        <p>
                                            <span className={'dataDescCardBold'}>Data Reference: </span>
                                            <span className={'dataDescCardItalic'}>sale_date</span>
                                        </p>
                                    </Card.Text>
                                </Card.Body>
                            </Card>
                            <Card border={'dark'}>
                                <Card.Header className='dataDescCardBold dataDescCardHeader'>Sale Price  </Card.Header>
                                <Card.Body>
                                    <Card.Text>
                                        <p>
                                            <span className={'dataDescCardBold'}>Example: </span>
                                            <span className={'dataDescCardItalic'}>40000.0</span>
                                        </p>
                                        <p>
                                            <span className={'dataDescCardBold'}>Description:</span> The last recorded
                                            sale of the property, in Dollar amount. If this field is
                                            <span className='dataDescCardItalic'> 1.0</span> it is not an error.
                                        </p>
                                        <p>
                                            <span className={'dataDescCardBold'}>Data Reference: </span>
                                            <span className={'dataDescCardItalic'}>sale_price</span>
                                        </p>
                                    </Card.Text>
                                </Card.Body>
                            </Card>
                            <Card border={'dark'}>
                                <Card.Header className='dataDescCardBold dataDescCardHeader'>Year Built </Card.Header>
                                <Card.Body>
                                    <Card.Text>
                                        <p>
                                            <span className={'dataDescCardBold'}>Example: </span>
                                            <span className={'dataDescCardItalic'}>2013-02-24</span>
                                        </p>
                                        <p>
                                            <span className={'dataDescCardBold'}>Description:</span> The year the
                                            property was built. <span className='dataDescCardBold'>Warning: </span>
                                            this may be Unknown or an Estimate.
                                        </p>
                                        <p>
                                            <span className={'dataDescCardBold'}>Data Reference: </span>
                                            <span className={'dataDescCardItalic'}>year_built</span>
                                        </p>
                                    </Card.Text>
                                </Card.Body>
                            </Card>
                            <Card border={'dark'}>
                                <Card.Header className='dataDescCardBold dataDescCardHeader'>Year Built Estimate</Card.Header>
                                <Card.Body>
                                    <Card.Text>
                                        <p>
                                            <span className={'dataDescCardBold'}>Example: </span>
                                            <span className={'dataDescCardItalic'}>Y or N</span>
                                        </p>
                                        <p>
                                            <span className={'dataDescCardBold'}>Description:</span> If the year built
                                            provided is an estimate.
                                        </p>
                                        <p>
                                            <span className={'dataDescCardBold'}>Data Reference: </span>
                                            <span className={'dataDescCardItalic'}>year_built_estimate</span>
                                        </p>
                                    </Card.Text>
                                </Card.Body>
                            </Card>
                            <Card border={'dark'}>
                                <Card.Header className='dataDescCardBold dataDescCardHeader'>Category </Card.Header>
                                <Card.Body>
                                    <Card.Text>
                                        <p>
                                            <span className={'dataDescCardBold'}>Example: </span>
                                            <span className={'dataDescCardItalic'}>Commercial</span>
                                        </p>
                                        <p>
                                            <span className={'dataDescCardBold'}>Description:</span> The type of
                                            property.
                                        </p>
                                        <p>
                                            <span className={'dataDescCardBold'}>Data Reference: </span>
                                            <span className={'dataDescCardItalic'}>category_code_description</span>
                                        </p>
                                    </Card.Text>
                                </Card.Body>
                            </Card>
                            <Card border={'dark'}>
                                <Card.Header className='dataDescCardBold dataDescCardHeader'>Recording Date </Card.Header>
                                <Card.Body>
                                    <Card.Text>
                                        <p>
                                            <span className={'dataDescCardBold'}>Example: </span>
                                            <span className={'dataDescCardItalic'}>2014-02-24</span>
                                        </p>
                                        <p>
                                            <span className={'dataDescCardBold'}>Description:</span> The date the
                                            property assessment was recorded.
                                        </p>
                                        <p>
                                            <span className={'dataDescCardBold'}>Data Reference: </span>
                                            <span className={'dataDescCardItalic'}>recording_date</span>
                                        </p>
                                    </Card.Text>
                                </Card.Body>
                            </Card>
                            <Card border={'dark'}>
                                <Card.Header className='dataDescCardBold dataDescCardHeader'>Zip Code </Card.Header>
                                <Card.Body>
                                    <Card.Text>
                                        <p>
                                            <span className={'dataDescCardBold'}>Example: </span>
                                            <span className={'dataDescCardItalic'}>19104</span>
                                        </p>
                                        <p>
                                            <span className={'dataDescCardBold'}>Description:</span> The zip code of
                                            property. Truncated to the first five digits.
                                        </p>
                                        <p>
                                            <span className={'dataDescCardBold'}>Data Reference: </span>
                                            <span className={'dataDescCardItalic'}>zip_code</span>
                                        </p>
                                    </Card.Text>
                                </Card.Body>
                            </Card>
                        </CardColumns>
                        <br/>
                    </div>
                </div>
                <div>
                    <h1>Data Handling</h1>
                    <p>
                        This site was put together by an experienced tenant organizer in the Philadelphia area. As this
                        information is public, data is only considered for omitting after a complaint
                        is sent to <a href="mailto:phl.landlord.spotter@gmail.com">phl.landlord.spotter@gmail.com</a>.
                    </p>
                </div>
                <div>
                    <h1>Data Concerns</h1>
                    <p>
                        If you have a concern or issue with this site or the data displayed on
                        it please email <a href="mailto:phl.landlord.spotter@gmail.com">phl.landlord.spotter@gmail.com</a>.
                    </p>
                </div>
            </Container>
        </div>
    )
}
