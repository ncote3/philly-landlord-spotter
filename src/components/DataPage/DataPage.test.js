import React from "react";
import { render, unmountComponentAtNode } from "react-dom";
import { act } from "react-dom/test-utils";
import DataPage from "./DataPage";
import pretty from "pretty";

let container = null;
beforeEach(() => {
    container = document.createElement('div');
    document.body.appendChild(container);
});

afterEach(() => {
    unmountComponentAtNode(container);
    container.remove();
    container = null;
})

it("renders correctly", () => {
    act(() => {
        render(
            <DataPage/>,
            container
        );
    });
    expect(container.innerHTML).toMatchSnapshot(pretty(container.innerHTML));
});