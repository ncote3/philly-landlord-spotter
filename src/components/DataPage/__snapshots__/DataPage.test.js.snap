// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly: <div class="DataPageWrapper">
  <div class="DataPageContainer container">
    <div class="DataPageJumbotron jumbotron">
      <h1>Data</h1>
      <p>An explanation on where this data came from and what it contains.</p>
    </div>
    <div class="DataPageSection whiteText">
      <div class="container">
        <h1>Data Source</h1>
        <p>In 2012 the City of Philadelphia began pursuing and enacting a public open data program through<a href="https://www.opendataphilly.org/"> OpenDataPhilly</a>. The data used in this project is provided by the aforementioned program, in their<a href="https://www.opendataphilly.org/dataset/opa-property-assessments"> Property Assessment </a>dataset. Publicly available, anyone can view or use the dataset without barrier.</p>
      </div>
    </div>
    <div class="DataPageSection">
      <div class="container">
        <div class="whiteText">
          <h1>Data Description</h1>
          <p>This dataset is comprised on completely public information. It does not contain any personal information about who lives in a property or the vacancy status of the properties.<br>The data used for this site contains the following for each property listed:</p>
        </div>
        <div>
          <div class="card-columns">
            <div class="card border-dark">
              <div class="dataDescCardBold dataDescCardHeader card-header">Owner 1 Name</div>
              <div class="card-body">
                <p class="card-text">
                <p><span class="dataDescCardBold">Example: </span><span class="dataDescCardItalic">John Doe</span></p>
                <p><span class="dataDescCardBold">Description:</span>The primary owner of a property.</p>
                <p><span class="dataDescCardBold">Data Reference: </span><span class="dataDescCardItalic">owner_1</span></p>
                </p>
              </div>
            </div>
            <div class="card border-dark">
              <div class="dataDescCardBold dataDescCardHeader card-header">Owner 2 Name</div>
              <div class="card-body">
                <p class="card-text">
                <p><span class="dataDescCardBold">Example: </span><span class="dataDescCardItalic">Jane Doe</span></p>
                <p><span class="dataDescCardBold">Description:</span>The secondary owner of a property. This is not always provided.</p>
                <p><span class="dataDescCardBold">Data Reference: </span><span class="dataDescCardItalic">owner_2</span></p>
                </p>
              </div>
            </div>
            <div class="card border-dark">
              <div class="dataDescCardBold dataDescCardHeader card-header">Street Address</div>
              <div class="card-body">
                <p class="card-text">
                <p><span class="dataDescCardBold">Example: </span><span class="dataDescCardItalic">123 Street Road</span></p>
                <p><span class="dataDescCardBold">Description:</span>The street address of the property.</p>
                <p><span class="dataDescCardBold">Data Reference: </span><span class="dataDescCardItalic">location</span></p>
                </p>
              </div>
            </div>
            <div class="card border-dark">
              <div class="dataDescCardBold dataDescCardHeader card-header">Geocoordinates</div>
              <div class="card-body">
                <p class="card-text">
                <p><span class="dataDescCardBold">Example: </span><span class="dataDescCardItalic">[40.003088,-75.108357]</span></p>
                <p><span class="dataDescCardBold">Description:</span>The latitude and longitude of a property.</p>
                <p><span class="dataDescCardBold">Data Reference: </span><span class="dataDescCardItalic">lat lng</span></p>
                </p>
              </div>
            </div>
            <div class="card border-dark">
              <div class="dataDescCardBold dataDescCardHeader card-header">Sale Price</div>
              <div class="card-body">
                <p class="card-text">
                <p><span class="dataDescCardBold">Example: </span><span class="dataDescCardItalic">40000.0</span></p>
                <p><span class="dataDescCardBold">Description:</span>The last recorded sale of the property, in Dollar amount. If this field is <span class="dataDescCardItalic"> 1.0</span> it is not an error.</p>
                <p><span class="dataDescCardBold">Data Reference: </span><span class="dataDescCardItalic">sale_price</span></p>
                </p>
              </div>
            </div>
            <div class="card border-dark">
              <div class="dataDescCardBold dataDescCardHeader card-header">Year Built</div>
              <div class="card-body">
                <p class="card-text">
                <p><span class="dataDescCardBold">Example: </span><span class="dataDescCardItalic">2013-02-24</span></p>
                <p><span class="dataDescCardBold">Description:</span>The year the property was built. <span class="dataDescCardBold">Warning:</span> this may be Unknown or an Estimate.</p>
                <p><span class="dataDescCardBold">Data Reference: </span><span class="dataDescCardItalic">year_built</span></p>
                </p>
              </div>
            </div>
            <div class="card border-dark">
              <div class="dataDescCardBold dataDescCardHeader card-header">Year Built Estimate</div>
              <div class="card-body">
                <p class="card-text">
                <p><span class="dataDescCardBold">Example: </span><span class="dataDescCardItalic">Y or N</span></p>
                <p><span class="dataDescCardBold">Description:</span>If the year built provided is an estimate.</p>
                <p><span class="dataDescCardBold">Data Reference: </span><span class="dataDescCardItalic">year_built_estimate</span></p>
                </p>
              </div>
            </div>
            <div class="card border-dark">
              <div class="dataDescCardBold dataDescCardHeader card-header">Category</div>
              <div class="card-body">
                <p class="card-text">
                <p><span class="dataDescCardBold">Example: </span><span class="dataDescCardItalic">Commercial</span></p>
                <p><span class="dataDescCardBold">Description:</span>The type of property.</p>
                <p><span class="dataDescCardBold">Data Reference: </span><span class="dataDescCardItalic">category_code_description</span></p>
                </p>
              </div>
            </div>
            <div class="card border-dark">
              <div class="dataDescCardBold dataDescCardHeader card-header">Recording Date</div>
              <div class="card-body">
                <p class="card-text">
                <p><span class="dataDescCardBold">Example: </span><span class="dataDescCardItalic">2014-02-24</span></p>
                <p><span class="dataDescCardBold">Description:</span>The date the property assessment was recorded.</p>
                <p><span class="dataDescCardBold">Data Reference: </span><span class="dataDescCardItalic">recording_date</span></p>
                </p>
              </div>
            </div>
            <div class="card border-dark">
              <div class="dataDescCardBold dataDescCardHeader card-header">Zip Code</div>
              <div class="card-body">
                <p class="card-text">
                <p><span class="dataDescCardBold">Example: </span><span class="dataDescCardItalic">19104</span></p>
                <p><span class="dataDescCardBold">Description:</span>The zip code of property. Truncated to the first five digits.</p>
                <p><span class="dataDescCardBold">Data Reference: </span><span class="dataDescCardItalic">zip_code</span></p>
                </p>
              </div>
            </div>
          </div><br>
        </div>
      </div>
    </div>
    <div class="DataPageSection whiteText">
      <div class="container">
        <h1>Data Handling</h1>
        <p>This site was put together by an experienced tenant organizer in the Philadelphia area. As this information is public, data is only considered for omitting after a complaint is sent to <a href="mailto:phl.landlord.spotter@gmail.com">phl.landlord.spotter@gmail.com</a>.</p>
      </div>
    </div>
    <div class="DataPageSection whiteText">
      <div class="container">
        <h1>Data Concerns</h1>
        <p>If you have a concern or issue with this site or the data displayed on it please email <a href="mailto:phl.landlord.spotter@gmail.com">phl.landlord.spotter@gmail.com</a>.</p>
      </div>
    </div>
  </div>
</div> 1`] = `"<div class=\\"DataPageWrapper\\"><div class=\\"DataPageContainer container\\"><div class=\\"DataPageJumbotron jumbotron\\"><h1>Data</h1><p>An explanation on where this data came from and what it contains.</p></div><div class=\\"DataPageSection whiteText\\"><div class=\\"container\\"><h1>Data Source</h1><p>In 2012 the City of Philadelphia began pursuing and enacting a public open data program through<a href=\\"https://www.opendataphilly.org/\\"> OpenDataPhilly</a>. The data used in this project is provided by the aforementioned program, in their<a href=\\"https://www.opendataphilly.org/dataset/opa-property-assessments\\"> Property Assessment </a>dataset. Publicly available, anyone can view or use the dataset without barrier.</p></div></div><div class=\\"DataPageSection\\"><div class=\\"container\\"><div class=\\"whiteText\\"><h1>Data Description</h1><p>This dataset is comprised on completely public information. It does not contain any personal information about who lives in a property or the vacancy status of the properties.<br>The data used for this site contains the following for each property listed:</p></div><div><div class=\\"card-columns\\"><div class=\\"card border-dark\\"><div class=\\"dataDescCardBold dataDescCardHeader card-header\\">Owner 1 Name</div><div class=\\"card-body\\"><p class=\\"card-text\\"><p><span class=\\"dataDescCardBold\\">Example: </span><span class=\\"dataDescCardItalic\\">John Doe</span></p><p><span class=\\"dataDescCardBold\\">Description:</span>The primary owner of a property.</p><p><span class=\\"dataDescCardBold\\">Data Reference: </span><span class=\\"dataDescCardItalic\\">owner_1</span></p></p></div></div><div class=\\"card border-dark\\"><div class=\\"dataDescCardBold dataDescCardHeader card-header\\">Owner 2 Name</div><div class=\\"card-body\\"><p class=\\"card-text\\"><p><span class=\\"dataDescCardBold\\">Example: </span><span class=\\"dataDescCardItalic\\">Jane Doe</span></p><p><span class=\\"dataDescCardBold\\">Description:</span>The secondary owner of a property. This is not always provided.</p><p><span class=\\"dataDescCardBold\\">Data Reference: </span><span class=\\"dataDescCardItalic\\">owner_2</span></p></p></div></div><div class=\\"card border-dark\\"><div class=\\"dataDescCardBold dataDescCardHeader card-header\\">Street Address</div><div class=\\"card-body\\"><p class=\\"card-text\\"><p><span class=\\"dataDescCardBold\\">Example: </span><span class=\\"dataDescCardItalic\\">123 Street Road</span></p><p><span class=\\"dataDescCardBold\\">Description:</span>The street address of the property.</p><p><span class=\\"dataDescCardBold\\">Data Reference: </span><span class=\\"dataDescCardItalic\\">location</span></p></p></div></div><div class=\\"card border-dark\\"><div class=\\"dataDescCardBold dataDescCardHeader card-header\\">Geocoordinates</div><div class=\\"card-body\\"><p class=\\"card-text\\"><p><span class=\\"dataDescCardBold\\">Example: </span><span class=\\"dataDescCardItalic\\">[40.003088,-75.108357]</span></p><p><span class=\\"dataDescCardBold\\">Description:</span>The latitude and longitude of a property.</p><p><span class=\\"dataDescCardBold\\">Data Reference: </span><span class=\\"dataDescCardItalic\\">lat lng</span></p></p></div></div><div class=\\"card border-dark\\"><div class=\\"dataDescCardBold dataDescCardHeader card-header\\">Sale Price</div><div class=\\"card-body\\"><p class=\\"card-text\\"><p><span class=\\"dataDescCardBold\\">Example: </span><span class=\\"dataDescCardItalic\\">40000.0</span></p><p><span class=\\"dataDescCardBold\\">Description:</span>The last recorded sale of the property, in Dollar amount. If this field is <span class=\\"dataDescCardItalic\\"> 1.0</span> it is not an error.</p><p><span class=\\"dataDescCardBold\\">Data Reference: </span><span class=\\"dataDescCardItalic\\">sale_price</span></p></p></div></div><div class=\\"card border-dark\\"><div class=\\"dataDescCardBold dataDescCardHeader card-header\\">Year Built</div><div class=\\"card-body\\"><p class=\\"card-text\\"><p><span class=\\"dataDescCardBold\\">Example: </span><span class=\\"dataDescCardItalic\\">2013-02-24</span></p><p><span class=\\"dataDescCardBold\\">Description:</span>The year the property was built. <span class=\\"dataDescCardBold\\">Warning:</span> this may be Unknown or an Estimate.</p><p><span class=\\"dataDescCardBold\\">Data Reference: </span><span class=\\"dataDescCardItalic\\">year_built</span></p></p></div></div><div class=\\"card border-dark\\"><div class=\\"dataDescCardBold dataDescCardHeader card-header\\">Year Built Estimate</div><div class=\\"card-body\\"><p class=\\"card-text\\"><p><span class=\\"dataDescCardBold\\">Example: </span><span class=\\"dataDescCardItalic\\">Y or N</span></p><p><span class=\\"dataDescCardBold\\">Description:</span>If the year built provided is an estimate.</p><p><span class=\\"dataDescCardBold\\">Data Reference: </span><span class=\\"dataDescCardItalic\\">year_built_estimate</span></p></p></div></div><div class=\\"card border-dark\\"><div class=\\"dataDescCardBold dataDescCardHeader card-header\\">Category</div><div class=\\"card-body\\"><p class=\\"card-text\\"><p><span class=\\"dataDescCardBold\\">Example: </span><span class=\\"dataDescCardItalic\\">Commercial</span></p><p><span class=\\"dataDescCardBold\\">Description:</span>The type of property.</p><p><span class=\\"dataDescCardBold\\">Data Reference: </span><span class=\\"dataDescCardItalic\\">category_code_description</span></p></p></div></div><div class=\\"card border-dark\\"><div class=\\"dataDescCardBold dataDescCardHeader card-header\\">Recording Date</div><div class=\\"card-body\\"><p class=\\"card-text\\"><p><span class=\\"dataDescCardBold\\">Example: </span><span class=\\"dataDescCardItalic\\">2014-02-24</span></p><p><span class=\\"dataDescCardBold\\">Description:</span>The date the property assessment was recorded.</p><p><span class=\\"dataDescCardBold\\">Data Reference: </span><span class=\\"dataDescCardItalic\\">recording_date</span></p></p></div></div><div class=\\"card border-dark\\"><div class=\\"dataDescCardBold dataDescCardHeader card-header\\">Zip Code</div><div class=\\"card-body\\"><p class=\\"card-text\\"><p><span class=\\"dataDescCardBold\\">Example: </span><span class=\\"dataDescCardItalic\\">19104</span></p><p><span class=\\"dataDescCardBold\\">Description:</span>The zip code of property. Truncated to the first five digits.</p><p><span class=\\"dataDescCardBold\\">Data Reference: </span><span class=\\"dataDescCardItalic\\">zip_code</span></p></p></div></div></div><br></div></div></div><div class=\\"DataPageSection whiteText\\"><div class=\\"container\\"><h1>Data Handling</h1><p>This site was put together by an experienced tenant organizer in the Philadelphia area. As this information is public, data is only considered for omitting after a complaint is sent to <a href=\\"mailto:phl.landlord.spotter@gmail.com\\">phl.landlord.spotter@gmail.com</a>.</p></div></div><div class=\\"DataPageSection whiteText\\"><div class=\\"container\\"><h1>Data Concerns</h1><p>If you have a concern or issue with this site or the data displayed on it please email <a href=\\"mailto:phl.landlord.spotter@gmail.com\\">phl.landlord.spotter@gmail.com</a>.</p></div></div></div></div>"`;
